common/lib/protocol-definitions/src/consensus.ts

```diff
@@ -150,7 +150,7 @@ export interface IProtocolState {
 }
 
 /**
- * @internal
+ * @alpha
```

Comment:
Any docs we can add while we're here?

Reply:
i'm not sure what this is or how to describe it